config/config.go

```diff
@@ -145,6 +153,13 @@ func (l *Local) Validate() []error {
 		errors = append(errors, err)
 	}
 
+	if l.IssueTracker != "" {
+		pattern, ok := originPatterns[l.IssueTracker]
+		if !ok || !pattern.MatchString(l.Origin) {
+			errors = append(errors, fmt.Errorf("origin is not valid for issue tracker: %s", l.IssueTracker))
```

Comment:
Could you make the err message slightly more informative by writing:
```
fmt.Sprintf("%s is not a valid origin for issue tracker %s", l.Origin, l.IssueTracker)
```

Reply:
resolved in 16833af